### ISSUE TITLE
continue cracking - captured before

### ISSUE THREAD

**Issue by wifiuk** [derv82/wifite2#22]
I seem to remember that there was an option to skip handshake capture in wifite if you had captured it before?

Can this be implemented in wifite2?

**Comment by derv82** [comment]
tl;dr yes soon

In Wifite V1, the behavior is to *completely ignore access points that have been cracked or handshake captured*. To re-attack an access point, the user has to specify `--recrack`.  Not specifying `--recrack` will actually *hide access points from the target list* (ugh).

I personally hate this feature since I'm testing and re-cracking the same access point over and over :(

*BUT* Wifite V1's behavior fits the narrative of a "set it and forget it" script. E.g. users run Wifite and target `all` access points, and the script only does what's needed -- not redundantly attacking the same target.

**Comment by wifiuk** [comment]
I don't think hiding the AP totally is great.
Thanks for all the work so far. This is my go to tool for WiFi work

**Comment by derv82** [comment]
Alright so to outline what needs to be done:

1. Still show cracked targets in the AP list
2. Show a `[!]` warning when attacking an AP we already cracked/captured.
3. WEP, WPS (pixie-dust & reaver), and WPA Handshake capture are **skipped** if we already have a handshake.
4. Tell user how to bypass this, using an option (`--dont-skip`?) or manually editing `cracked.txt`
5. If it's a handshake but the key is unknown, tell user to use the `--crack` option to attempt to crack an already-captured handshake.

**Comment by evrial** [comment]
What's the current status of this feature? Also would be nice to add cracked wpa passwords from hashcat and skip those networks in future.

**Comment by rpodgorny** [comment]
a friendly bump. is this still being worked on? what can i do to help finish this feature?
